### PR TITLE
ios: fix callkit calls via NSE

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/Shared/Views/Call/CallController.swift [hunk 1]
```diff
@@ -142,45 +142,57 @@ class CallController: NSObject, CXProviderDelegate, PKPushRegistryDelegate, Obse
...
-        logger.debug("CallController: initializing chat")
+        if AppChatState.shared.value == .stopped {
+            self.reportExpiredCall(payload: payload, completion)
+            return
+        }
         if (!ChatModel.shared.chatInitialized) {
-            initChatAndMigrate(refreshInvitations: false)
+            logger.debug("CallController: initializing chat")
+            do {
+                try initializeChat(start: true, refreshInvitations: false, withChatData: false)
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Call/CallController.swift:161]: doesn't compile with extra argument error
```suggestion
                try initializeChat(start: true, refreshInvitations: false)
```

File: apps/ios/Shared/Model/SuspendChat.swift [hunk 2]
```diff
@@ -134,20 +136,33 @@ func initChatAndMigrate(refreshInvitations: Bool = true) {
     }
 }
 
-func startChatAndActivate(dispatchQueue: DispatchQueue = DispatchQueue.main, _ completion: @escaping () -> Void) {
+func startChatForCall() {
```
Comment (spaced4ndy) [inline apps/ios/Shared/Model/SuspendChat.swift:139]: add a comment that the main difference with startChatAndActivate is that it doesn't wait for NSE? If I understand correctly. A bit hard to see behind different calls to activateChat(), via activate() in startChatAndActivate which does the same but with completion()

File: apps/ios/SimpleX.xcodeproj/xcshareddata/xcschemes/SimpleX (iOS).xcscheme [hunk 3]
```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```
Comment (spaced4ndy) [inline apps/ios/SimpleX.xcodeproj/xcshareddata/xcschemes/SimpleX (iOS).xcscheme:44]: don't forget to revert

Comment (epoberezkin) [reply apps/ios/SimpleX.xcodeproj/xcshareddata/xcschemes/SimpleX (iOS).xcscheme:44]: better keep as is

**Pull request merged**